nais/nais-dev.yaml

```diff
@@ -38,10 +38,10 @@ spec:
     - "https://aktivitetskrav-backend.intern.dev.nav.no"
   resources:
     limits:
-      memory: 400Mi
+      memory: 800Mi
```

Comment:
Er det noe spesielt grunn at vi trenger vi å øke memory i `dev`?

Reply:
Det er et eksperiment vi ville gjøre. Vi har en teori om at årsaken til restarts kan være at den max'er ut det som er memory limit på pod'en. Og så for den SIGTERN request fra kubernetes og at det trigger restart'ene.
(hva limit bør hver, og hvorfor MaxRAMPercentage ikke holder den innenfor mangler vi forklaring på foreløbig.)


![image](https://github.com/user-attachments/assets/e33b9bd2-d104-4a62-91ba-357fd56c0f5c)